OptimizelySDKCore/OptimizelySDKCoreTests/OptimizelyTest.m

```diff
@@ -267,19 +266,6 @@ - (void)testOptimizelyTrackWithEventOfNoExperiment {
     [loggerMock stopMocking];
 }
 
```

Comment:
Can we add a test for the track notification listener?

Reply:
Test added for track notification listener